src/cc/usdt.h

```diff
@@ -156,6 +156,9 @@ class ArgumentParser_x64 : public ArgumentParser {
     REG_SI,
     REG_DI,
     REG_BP,
+#ifdef REG_SP
+#undef REG_SP
+#endif
```

Comment:
This seems a bit hacky.

Reply:
Yes, I agree with you, it is more a matter of necessity.
Maybe we can suggest to modify the macro name REG_SP in libc-dev, 
but it is not so easy, so just undef it now to make it works.

Reply:
Sorry for the late reply.  Thanks for your suggestion.

The sys/ucontext.h of LoongArch glibc has only an internal version now, I submitted a bug to the related engineer to modify it as LOONGARCH_REG_SP before upstream, so it is not necessary to do this change in src/cc/usdt.h, I will resubmit this patch until LoongArch glibc is settled.